Data/HashMap/Base.hs

```diff
@@ -1,6 +1,10 @@
 {-# LANGUAGE BangPatterns, CPP, DeriveDataTypeable, MagicHash #-}
 {-# OPTIONS_GHC -fno-full-laziness -funbox-strict-fields #-}
 
+#if defined(__GLASGOW_HASKELL__)
```

Comment:
You can remove this ifdef. We only need to support the last 3 major releases (so that's back to GHC 7.4) and this pragma existed since 7.2. There are no other compilers than GHC. :)